src/ExportMap.js

```diff
@@ -530,30 +530,50 @@ ExportMap.parse = function (path, content, context) {
 
     // This doesn't declare anything, but changes what's being exported.
     if (n.type === 'TSExportAssignment') {
-      const moduleDecls = ast.body.filter((bodyNode) =>
-        bodyNode.type === 'TSModuleDeclaration' && bodyNode.id.name === n.expression.name
+      const exportedName = n.expression.name
+      const declTypes = [
+        'VariableDeclaration',
+        'ClassDeclaration',
+        'TSDeclareFunction',
+        'TSEnumDeclaration',
+        'TSTypeAliasDeclaration',
+        'TSInterfaceDeclaration',
+        'TSAbstractClassDeclaration',
+        'TSModuleDeclaration',
+      ]
+      const exportedDecls = ast.body.filter(({ type, id, declarations }) => 
+        declTypes.includes(type) && 
+        (id && id.name === exportedName || declarations.find(d => d.id.name === exportedName))
```

Comment:
Hey! Here `declarations` is undefined unless `type === 'VariableDeclarations'` so it crashes the linter in some cases. I'll create another PR to fix it.

Reply:
Thanks!